docs/plugin-transform-runtime.md

```diff
@@ -4,37 +4,37 @@ title: babel-plugin-transform-runtime
 sidebar_label: transform-runtime
 ---
 
-NOTE: Instance methods such as `"foobar".includes("foo")` will not work since that would require modification of existing built-ins (Use [`@babel/polyfill`](http://babeljs.io/docs/usage/polyfill) for that).
+A plugin that enables the re-use of Babel's injected helper code to save on codesize.
 
-## Why?
-
-Babel uses very small helpers for common functions such as `_extend`. By default this will be added to every file that requires it. This duplication is sometimes unnecessary, especially when your application is spread out over multiple files.
-
-This is where the `@babel/plugin-transform-runtime` plugin comes in: all of the helpers will reference the module `@babel/runtime` to avoid duplication across your compiled output. The runtime will be compiled into your build.
-
-Another purpose of this transformer is to create a sandboxed environment for your code. If you use [@babel/polyfill](http://babeljs.io/docs/usage/polyfill/) and the built-ins it provides such as `Promise`, `Set` and `Map`, those will pollute the global scope. While this might be ok for an app or a command line tool, it becomes a problem if your code is a library which you intend to publish for others to use or if you can't exactly control the environment in which your code will run.
-
-The transformer will alias these built-ins to `core-js` so you can use them seamlessly without having to require the polyfill.
-
-See the [technical details](#technical-details) section for more information on how this works and the types of transformations that occur.
+> NOTE: Instance methods such as `"foobar".includes("foo")` will not work since that would require modification of existing built-ins (you can use [`@babel/polyfill`](polyfill.md) for that).
 
 ## Installation
 
-**NOTE - Production vs. development dependencies**
-
-In most cases, you should install `@babel/plugin-transform-runtime` as a development dependency (with `--save-dev`).
+Install it as development dependency.
 
 ```sh
 npm install --save-dev @babel/plugin-transform-runtime
 ```
 
-and `@babel/runtime` as a production dependency (with `--save`).
+and [`@babel/runtime`](runtime.md) as a production dependency (since it's for the "runtime").
```

Comment:
I'm wondering - can this somehow be made obvious? Like maybe we should validate it? This thing is really confusing for newcomers and it's easy to miss installing this

Reply:
Logan was thinking about what we can do here but not sure what we can do for v7.